spec/integ/matrix-client-event-timeline.spec.ts

```diff
@@ -649,13 +649,40 @@ describe("MatrixClient event timelines", function() {
                     };
                 });
 
-            return Promise.all([
-                expect(client.getEventTimeline(timelineSet, EVENTS[0].event_id)).resolves.toBeUndefined(),
-                httpBackend.flushAllExpected(),
-            ]);
+            // getEventTimeline -> thread.fetchInitialEvents
+            httpBackend.when("GET", "/rooms/!foo%3Abar/relations/" +
+                encodeURIComponent(THREAD_ROOT.event_id) + "/" +
```

Comment:
There are some lints in CI: https://github.com/matrix-org/matrix-js-sdk/runs/8077711103?check_suite_focus=true

But I don't see them locally even after re-installing `node_modules` to ensure correct versions.
```
$ yarn lint
yarn run v1.22.18
$ yarn lint:types && yarn lint:js
$ tsc --noEmit
$ eslint --max-warnings 0 src spec
✨  Done in 21.60s.
```

---

This is also the same pattern we use in the existing tests here but don't appear because they're not in the diff so will need to refactor this.

Reply:
Those are shown if you use `tsc --strict` - hence being reported by the `Typescript Strict Error Checker`

Reply:
Can probably merge `develop` to resolve all of these unrelated errors now that https://github.com/matrix-org/matrix-js-sdk/pull/2835 fixed them up